bundles/net.tourbook/src/net/tourbook/tour/DialogQuickEdit.java

```diff
@@ -536,7 +536,7 @@ private void createUI_141_Weather_Description(final Composite parent) {
                //
                // SWT.DEFAULT causes lot's of problems with the layout therefore the hint is set
                //
-               .hint(_pc.convertWidthInCharsToPixels(80), _pc.convertHeightInCharsToPixels(6))
+               .hint(_pc.convertWidthInCharsToPixels(80), _pc.convertHeightInCharsToPixels(2))
```

Comment:
I decided to reduce it because for a "Quick Edit" dialog, the weather description field is HUGE, it's bigger than the tour description field!! and that didn't seem to make sense when I thought about it
![image](https://user-images.githubusercontent.com/25235665/222453762-fe7a7e1f-c8ab-4696-9114-8ab0fba8c3d2.png)



I will use the number of lines specified in the Tour Editor slideout

Reply:
> I decided to reduce it because for a "Quick Edit" dialog, the weather description field is HUGE, 

I increased the number of lines because when you are doing a tour for some hours, each hour needs one line and I had to scroll too much

> it's bigger than the tour description field!! 

This is true and false, it depends on the editor window size and which parts in the tour editor are collapsed. 
This field has a default height which the user can select **and** this field **is grabbing all vertical space** which is not used

Reply:
> I increased the number of lines because when you are doing a tour for some hours, each hour needs one line and I had to scroll too much

That means you're using the weather feature ? 😄  👍 
> 

> This is true and false, it depends on the editor window size and which parts in the tour editor are collapsed. This field has a default height which the user can select **and** this field **is grabbing all vertical space** which is not used

I think it's better that the user can decide what height is best for them since we all use MT differently

Reply:
> > I increased the number of lines because when you are doing a tour for some hours, each hour needs one line and I had to scroll too much
> 
> That means you're using the weather feature ? 😄 👍

Sure, it's easy, you have to setup it once in easy import and then never care about it